### PR TITLE
Add Zopim location and events

### PR DESCRIPTION
**Pull request by danielbreves**
:v:

This should help you guys to test the new APIs and events.

screen shot 2016-08-17 at 5 30 58 pm

/cc @zendesk/vegemite @yatishb @anilande @edodat @Mehul-Porwal

### CODE REVIEW

**Comment by Mehul-Porwal** [comment]
@danielbreves Thanks for your efforts. @zopim/pando has already created an app for testing the data and events APIs. We will investigate if we can use anything from this app for our testing(manual&automated).

**Comment by sandlerr** [comment]
:+1:

**Comment by danielbreves** [comment]
cc @zendesk/pando

File: src/javascripts/app.js [hunk 1]
```diff
@@ -216,7 +216,11 @@ var App = {
   'notification.repl',
   'message.repl',
   'window.resize',
-  'window.scroll'
+  'window.scroll',
+  'channel.chat.started',
```
Comment (yatishb) [inline src/javascripts/app.js]: This event has now been changed to ```channel.chat.start``` and ```channel.start.end```

**Comment by yatishb** [comment]
👍

**Pull request merged**